services/deployment-agent/.env.config

```diff
@@ -0,0 +1,8 @@
+# defines which configuration file to use
```

Comment:
Minor: Should we remove the '.' prefix in these files and just call them ``env.config``  (i.e. environment configuration)? 
The files with prefixed with . denote that they are hidden (e.g. when they are files produced temporarily or a hidden configuration that is not that relevant as .gitignore) but this makes no sense here, right? . In the case of `.env` is some legacy that I still do not fully make sense